mockProject/Autoload.ts

```diff
@@ -0,0 +1,6 @@
+// this file will have dynamic content from our test infrastructure
```

Comment:
Writing nit: "This file will be written to by ts2gd's tests."

Reply:
Well yes, but no. We have it here so that the fs can see it but the tests aren't writing to it. (We return different content to tsc).